### PR TITLE
feat(server): add steam_webApiDomain convar

### PR DESCRIPTION
**Pull request by kasuganosoras**
Fix this issue: #1417

### CODE REVIEW

File: ext/ui-build/data/root.html [hunk 1]
```diff
@@ -80,7 +80,7 @@
 		const frame = document.createElement('iframe');
 		frame.name = frameName;
 		frame.style.visibility = 'hidden';
-		frame.allow = 'microphone *; camera *;';
+		frame.allow = 'microphone *; camera *; midi *;';
```
Comment (FabianTerhorst) [inline ext/ui-build/data/root.html:83]: This change seems unrelated to the steam api change. Please don't add it in the same pull request.

**Review by FabianTerhorst — CHANGES_REQUESTED** [review]
Please remove the midi change from the pull request and modfy the commit message to fit our standard.

Example:
```
feat(server): add steam_webApiDomain convar
```

**Comment by kasuganosoras** [comment]
> Please remove the midi change from the pull request and modfy the commit message to fit our standard.
> 
> Example:
> 
> ```
> feat(server): add steam_webApiDomain convar
> ```

OK, I removed it.

**Comment by kasuganosoras** [comment]
> > > Please remove the midi change from the pull request and modfy the commit message to fit our standard.
> > > Example:
> > > ```
> > > feat(server): add steam_webApiDomain convar
> > > ```
> > 
> > 
> > OK, I removed it.
> 
> Thanks, now only the commit squashing is required with commit text editing to `feat(server): add steam_webApiDomain convar`. When you don't know how to do it check e.g. https://docs.github.com/en/desktop/managing-commits/squashing-commits-in-github-desktop

Sorry, I messed up the commit squashing, so I resynchronized the branch and commit the code again

**Review by FabianTerhorst — APPROVED** [review]
👍
